src/renderer/pages/project/script_editor.vue

```
@@ -88,7 +88,7 @@
                 />
               </div>
               <div class="px-4 pt-2">
-                <BeatSelector @emitBeat="(beat) => addBeat(beat, index)" buttonKey="insert" />
+                <BeatSelector @emitBeat="(beat) => addBeat(beat, index)" buttonKey="insert" :isPro="globalStore.userIsPro"  />
```

ESLint

[error] Replace `·@emitBeat="(beat)·=>·addBeat(beat,·index)"·buttonKey="insert"·:isPro="globalStore.userIsPro"` with `⏎··················@emitBeat="(beat)·=>·addBeat(beat,·index)"⏎··················buttonKey="insert"⏎··················:isPro="globalStore.userIsPro"⏎··············`

(prettier/prettier)